modules/apps/ratings/ratings-taglib/src/main/resources/META-INF/resources/page.jsp

```diff
@@ -17,6 +17,11 @@ String type = GetterUtil.getString((String)request.getAttribute("liferay-ratings
 	<link href="<%= PortalUtil.getStaticResourceURL(request, PortalUtil.getPathProxy() + application.getContextPath() + "/css/main.css") %>" rel="stylesheet" type="text/css" />
 </liferay-util:html-top>
 
+<c:if test="<%= themeDisplay.isIsolated() %>">
```

Comment:
@boton suggested to load the css directly in the `ContentRatingsFragmentRenderer`, which must be the fragment that loads this. Do you know how I should do that @victorg1991 ?? Or is the current fix better?

Reply:
@victorg1991 Isn't there some simple way to load a CSS around `modules/apps/fragment/fragment-impl/src/main/java/com/liferay/fragment/internal/renderer/ContentRatingsFragmentRenderer.java`? 

I only discover [this one](https://github.com/liferay/liferay-portal/blob/b808cb100001223008944254d5da1364de64c622/modules/apps/fragment/fragment-impl/src/main/java/com/liferay/fragment/internal/renderer/FragmentEntryFragmentRenderer.java#L252-L254) but too complex for our case.

Reply:
Sorry for the assault but I was just checking the PR.

Reply:
Yeah, I think we can do that, but I think the result will be more or less the same :thinking: Since we need to check that the same conditions apply, isn't it?

Reply:
That is true, we will need to check both cases in `ContentRatingsFragmentRenderer`, and cannot remove the `html-top` in the .jsp, because when rendering the taglib `liferay-ratings:rating`, we don't go through the FragmentRenderer.  
So let's go with this option.